src/index.html

```diff
@@ -12,7 +12,45 @@
       href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__img"
+        src="./images/imac.jpeg"
+        alt="imac"
+      />
+      <h2 class="product-card__title">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="product-card__text">Product code: 195434</p>
+
+      <div class="product-card__grade">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="product-card__reviews">Reviews: 5</p>
+      </div>
+      <div class="product-card__price-container">
+        <p class="product-card__price-title">Price:</p>
+        <p class="product-card__price">$2,199</p>
+      </div>
+      <button
```

Comment:
Add the blank line between the blocks
```suggestion
      </div>
      
      <div class="product-card__price-container">
        <p class="product-card__price-title">Price:</p>
        <p class="product-card__price">$2,199</p>
      </div>
      
      <button
```

Reply:
Done